common/lib/xmodule/xmodule/js/src/html/edit.coffee

```diff
@@ -83,6 +83,7 @@ class @HTMLEditingDescriptor
         image_advtab: true,
         # We may want to add "styleselect" when we collect all styles used throughout the LMS
         toolbar: "formatselect | fontselect | bold italic underline forecolor wrapAsCode | " +
+          "alignleft aligncenter alignright alignjustify | " +
```

Comment:
@fysheets Do these options apply to the text in the HTML editor as well as the image?

Reply:
Yes they will -- we can run through an example together if that's helpful.

Reply:
<img width="1347" alt="screen shot 2018-04-10 at 5 01 59 pm" src="https://user-images.githubusercontent.com/9357438/38583435-01c6c074-3ce1-11e8-9ebe-84b47838c43a.png">

<img width="973" alt="screen shot 2018-04-10 at 5 00 37 pm" src="https://user-images.githubusercontent.com/9357438/38583378-d017bf1a-3ce0-11e8-9106-d4c0b400c027.png">

Here's a quick screenshot of the toolbar + what the alignment looks like for each type. I'll add to the ticket as well.